include/nbl/asset/IImage.h

```diff
@@ -558,14 +558,14 @@ class IImage : public IDescriptor
 			EL_UNDEFINED = 0,
 			EL_GENERAL = 1,
 			EL_COLOR_ATTACHMENT_OPTIMAL = 2,
-			EL_DEPTH_STENCIL_ATTACHMENT_OPTIMAL = 3,
-			EL_DEPTH_STENCIL_READ_ONLY_OPTIMAL = 4,
 			EL_SHADER_READ_ONLY_OPTIMAL = 5,
 			EL_TRANSFER_SRC_OPTIMAL = 6,
 			EL_TRANSFER_DST_OPTIMAL = 7,
 			EL_PREINITIALIZED = 8,
-			EL_DEPTH_READ_ONLY_STENCIL_ATTACHMENT_OPTIMAL = 1000117000,
-			EL_DEPTH_ATTACHMENT_STENCIL_READ_ONLY_OPTIMAL = 1000117001,
+			EL_DEPTH_ATTACHMENT_OPTIMAL = 1000241000,
+			EL_DEPTH_READ_ONLY_OPTIMAL = 1000241001,
+			EL_STENCIL_ATTACHMENT_OPTIMAL = 1000241002,
+			EL_STENCIL_READ_ONLY_OPTIMAL = 1000241003,
```

Comment:
@Erfan-Ahmadi now I need to go through every function that has anything to do with these

Reply:
done